src/CppAst/CppTokenUtil.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics;
 using System.Linq.Expressions;
 using System.Text;
+using System.Threading;
```

Comment:
Do we really need this? I don't see threading being used in this file.

Reply:
I don't think so, I'm not sure why that ended up there. I'll remove it and if I get no errors, leave it removed.